rhombus-lib/rhombus/private/amalgam/port.rkt

```diff
@@ -153,6 +173,11 @@
     [() (open-output-bytes)]
     [(name) (open-output-bytes name)]))
 
+(define/arity (Port.Output.open_file path #:exists [exists 'error])
+  #:primitive (open-output-file)
+  #:static-infos ((#%call-result #,(get-output-port-static-infos)))
+  (open-output-file path #:exists (->ExistsFlag exists)))
```

Comment:
This needs to check that the result of `->ExistsFlag` is non-`#f` and raise an annotation error otherwise (but also check `path` before raising the annotation error).

Reply:
Fixed, thanks!